app/AppComponents.scala

```diff
@@ -35,7 +34,7 @@ class AppComponents(context: Context, config: Config)
   // to request them. Seems to be a timing bug...
   //
   // This should only be a temporary fix @ 2016/02/09
-  Permissions.list(PermissionsUser("preload@permissions"))
+//  Permissions.listPermission(PermissionsUser("preload@permissions"))
```

Comment:
I think this can go now as it seems to work for me but I'm not sure what the best way to test this is.

Reply:
I'd be tempted to leave it in, given that it's been there for 7 years. It sounds like it could cause problems if a new permission is added, based on that comment, and its not causing any problems being there. 

I feel like the risk of the same obscure problem emerging if we add a permission in the future is higher than the value of removing it.

Reply:
Origin PR doesn't have much on the Permissions issue it references: https://github.com/guardian/tagmanager/pull/149

Reply:
Yes exactly, don't see this problem in other services that use the permissions library. Also in the original PR, this line of code was part of `PermissionsLoadModule`, which is now gone. I'm tempted to remove this, as leaving this in would require refactor due to the permissions version upgrade. I'll remove and test this in CODE.